src/fsharp/FSharp.Core/printf.fs

```diff
@@ -824,13 +827,17 @@ module internal PrintfImpl =
 
             let useZeroWidth = isPadWithZeros spec.Flags
             let opts = 
-                let o = Microsoft.FSharp.Text.StructuredPrintfImpl.FormatOptions.Default
-                let o =
+                Microsoft.FSharp.Text.StructuredPrintfImpl.FormatOptions.Default
+                |> fun o ->
                     if useZeroWidth then { o with PrintWidth = 0} 
```

Comment:
Just use ``let`` bindings - that's normal style in FSharp.Core. If you like give all the ``o`` different names.